### ISSUE TITLE
gfortran support

### ISSUE THREAD

**Issue by bricerebsamen** [rizsotto/Bear#201]
I am hoping to use bear to parse the build for OpenBlas, then write a python script to convert the json to a bunch bazel rules.
OpenBlas uses gfortran to compile lapack.
for now I am using `make | grep gfortran > fortran_commands.txt` to get the gfortran commands. It'd be great to have gfortran support in bear though.

**Comment by rizsotto** [comment]
Hey @bricerebsamen , thanks for your report. Some previous version had the ability to dump all captured commands into a JSON file. This is gone now. I'm working on a new version which will give more insights into the build process (what was the exit code of the commands, how long did it take, etc...). But it's still in progress... Till then I would recommend to modify the `bear` executable (comment out the filter methods) as a quick workaround for your problem.

**Comment by bricerebsamen** [comment]
thanks. I managed to capture the fortran command (that was a bit more involved than what you suggested...). Unfortunately that did not help me very much because the makefile for OpenBlas also builds some binaries (even when asked to just build the library). So I had to look at the call to `ar` to keep only the objects that were needed. At the end of the day I ended up taking a fairly manual approach...
but bear did help me a lot for the c++ part of the library :)
